go.sum

```diff
@@ -25,6 +25,42 @@ github.com/armon/go-radix v1.0.0/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgI
 github.com/aws/aws-sdk-go v1.31.9/go.mod h1:5zCpMtNQVjRREroY7sYe8lOMRSxkhG6MZveU8YkpAk0=
```

Comment:
Why is the first version this required and not purged?

Reply:
It's use by validate_cassettes_test.go